app/models/blacklight_config.rb

```diff
@@ -24,7 +24,8 @@ def configurations
       DefaultConfiguration.new(configuration),
       SearchFieldConfiguration.new(configuration),
       ShowActions.new(configuration),
-      ThumbnailConfiguration.new(configuration)
+      ThumbnailConfiguration.new(configuration),
+      Gallery.new(configuration)
```

Comment:
I'm ambivalent, because of their namespacing I'm even tempted to kill Configuration from the other objects.